package.json

```diff
@@ -127,10 +127,8 @@
     "apollo-link-http": "^1.5.16",
     "apollo-link-ws": "^1.0.19",
     "assume": "^2.1.0",
-    "buffered-async-iterator": "^1.0.0",
     "builtin-modules": "^3.1.0",
     "c8": "^6.0.0",
-    "cli-spinners": "^2.3.0",
```

Comment:
/cc @helfi92  :)

Reply:
is this actually not unused?

Reply:
It's unused -- I mentioned it in https://github.com/taskcluster/taskcluster/pull/2699#pullrequestreview-396540767 but I think it got missed :)